web/src/server/api/routers/generations/exportQuery/transforms/transformStreamToCsv.ts

```diff
@@ -1,51 +1,83 @@
 import { Transform, type TransformCallback } from "stream";
 
-import { usdFormatter } from "@/src/utils/numbers";
-
-import type { ObservationView } from "@prisma/client";
+import { type ObservationViewWithScores } from "@/src/server/api/routers/generations/getAllQuery";
+import { formatIntervalSeconds, intervalInSeconds } from "@/src/utils/dates";
 
 export function transformStreamToCsv(): Transform {
   let isFirstChunk = true;
 
   return new Transform({
     objectMode: true,
     transform(
-      row: ObservationView,
+      row: ObservationViewWithScores,
       encoding: BufferEncoding,
       callback: TransformCallback,
     ): void {
       if (isFirstChunk) {
         // Output the header if it's the first chunk
         const csvHeader = [
-          "traceId",
+          "id",
           "name",
-          "model",
+          "traceId",
+          "traceName",
           "startTime",
           "endTime",
-          "cost",
-          "prompt",
-          "completion",
+          "timeToFirstToken",
+          "scores",
+          "latency",
+          "latencyPerToken",
+          "inputCost",
+          "outputCost",
+          "totalCost",
+          "level",
+          "statusMessage",
+          "model",
+          "promptTokens",
+          "completionTokens",
+          "totalTokens",
+          "input",
+          "output",
           "metadata",
+          "promptName",
+          "promptVersion",
         ];
 
         this.push(csvHeader.join(",") + "\n");
 
         isFirstChunk = false;
       }
-
       // Convert the generation object to a CSV line and push it
       const csvRow = [
```

Comment:
can we also include parent_observation_id, type, modelParameters, completion_start_time

Reply:
... and do the same for the other export?